broadcast/index.md

```diff
@@ -31,7 +31,11 @@ and support for Trino. We dive into the details and have a look at a demo.</dd>
 <dd><a href="https://www.linkedin.com/in/manfredmoser">Manfred Moser</a> talks
 about his recent work to improve Trino packaging and demos how to create your 
 own RPM, tarball, and container image.</dd>
-
+<dt>6 May 2025: Trino Community Broadcast 74 - One tool to proxy them all</dt>
+<dd><a href="https://www.linkedin.com/in/jordanzimmerman/">Jordan Zimmermann</a>
+and <a href="https://www.linkedin.com/in/cristian-osiac/">Cristian Osiac</a>
```

Comment:
Include Pablo too please: https://www.linkedin.com/in/pablo-arteaga-20b547101/

Reply:
Done